pkg/types/openstack/defaults/platform.go

```diff
@@ -1,9 +1,37 @@
 package defaults
 
 import (
+	"fmt"
+
+	"github.com/apparentlymart/go-cidr/cidr"
+
+	"github.com/openshift/installer/pkg/types"
 	"github.com/openshift/installer/pkg/types/openstack"
 )
 
+// Defaults for the openstack platform.
+const (
+	APIVIP = ""
```

Comment:
why they public? If they need to be public this needs comments.

Reply:
Oops, these are leftovers from previous iterations. They're not actually used anywhere. Going to remove them.